EIPS/eip-FOCIL.md

```diff
@@ -32,27 +32,27 @@ FOCIL is a simple committee-based design improving upon previous IL mechanisms o
 
 #### Timeline
 
-A set of validators is selected from the beacon committee to become IL committee members for `slot N`.
+A set of validators is selected from the beacon committee to become IL committee members for `slot N+1`.
```

Comment:
My preference is to define the IL committee for `N` to enforce the block proposer for `N+1`. I find this way of expressing it less confusing. The other option you suggested is for the `N+1` IL committee to enforce the block proposer for `N+1`, but the `N+1` IL committee would perform its duties at `N`. I find this expression more interleaving. I don't have a strong preference, but we should be consistent here.

```suggestion
A set of validators is selected from the beacon committee to become IL committee members for `slot N`.
```

Reply:
Good point, change the original `Slot N, t ==6` bullet to use "IL committee for N" accordingly.